userspace/falco/app/actions/init_inspectors.cpp

```diff
@@ -87,8 +87,12 @@ static void init_syscall_inspector(falco::app::state& s, std::shared_ptr<sinsp>
 	}
 
 	//
-	// If required, set the snaplen
+	// If required, set the snaplen.
+	// In case both config and CLI options are specified, CLI takes precedence.
 	//
+	if(s.config->m_falco_libs_snaplen != 0) {
+		inspector->set_snaplen(s.config->m_falco_libs_snaplen);
+	}
 	if(s.options.snaplen != 0) {
 		inspector->set_snaplen(s.options.snaplen);
```

Comment:
Do we want to add the `attention this is deprectated and will be removed in..." log here? Like this one: https://github.com/falcosecurity/falco/pull/3329/files#diff-e1e4257c43745e29751e72ffa753fd44ec931b07b1bec5035f4d1b04d061848cR100